### ISSUE TITLE
Accessibility Audit: Screen readers do not indicate an image is displayed in Openseadragon canvas area 

### ISSUE THREAD

**Issue by brittnylapierre** [ProjectMirador/mirador#3881]
Description: An accessibility audit of Mirador was performed. During the audit it was found that additional information should be made available by the viewer about the nature of the content visible in the canvas area for users with screen readers; For example, telling them that this section renders the content of the <canvas index> item in the IIIF manifest titled <manifest label>. It has the label, <canvas label>. This section typically renders either an image, video, or audio file. 

Supporting Comments:
- LIMITED SIGHT – Windows 10/Windows Narrator/EDGE - Nothing under Digitized viewer heading (image section) was readable by Narrator except zoom and navigation buttons.
